index.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;700&display=swap" rel="stylesheet">
+    <title>Samuel Felice | Full Stack Developer</title>
+</head>
+
+<body>
+    <header>
+        <a href="#" class="logo">My Logo</a>
+        <nav class="nav-icon">
+            <div class="icon"></div>
+            <div class="icon"></div>
+            <div class="icon"></div>

```

Comment:
- [ ]  OPTIONAL
- To also achieve this, you can download the image by grouping them from Figma in order to reduce many line of code :+1: